6486127/test
# Layout

# Pull Request: Setting up environment and creating layout  
## Description
This pull request includes the following changes:

1. Setting up environment and Gulp
2. Creating layout based on Figma design
3. Adding form validation
4. Implementing responsive design  

## Changes Made
1. Environment setup and Gulp configuration
2. Created layout based on [Figma design](https://www.figma.com/file/wfDlafwIsiExCjO0BHOSS3/login_signup_test-(Copy)-(Copy)?type=design&node-id=0-2&mode=design&t=CXk5gsdJ4QZLkaZa-0)
3. Added form validation to ensure all inputs are filled in correctly before submitting
4. Implemented responsive design to ensure it looks good on all screen sizes  

## How to test
1. Clone the repository and check out this branch
2. Run **npm install** to install dependencies
3. Run **gulp** to start watching for changes
4. Open the **index.html** file in a browser and test the form
